src/derivable/atom.spec.ts

```diff
@@ -40,6 +40,27 @@ describe('derivable/atom', () => {
         });
     });
 
+    describe('#value', () => {
```

Comment:
This test can be moved to `derivable.spec.ts` behind an `if (isAtom(value$))` guard. This way it is also applied to Lenses.